modules/base/rendering/screenspacetimevaryingimageonline.cpp

```diff
@@ -0,0 +1,234 @@
+/*****************************************************************************************
+ *                                                                                       *
+ * OpenSpace                                                                             *
+ *                                                                                       *
+ * Copyright (c) 2014-2025                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/base/rendering/screenspacetimevaryingimageonline.h>
+
+#include <openspace/documentation/documentation.h>
+#include <openspace/documentation/verifier.h>
+#include <openspace/engine/globals.h>
+#include <openspace/json.h>
+#include <openspace/util/timemanager.h>
+#include <ghoul/filesystem/filesystem.h>
+#include <ghoul/io/texture/texturereader.h>
+#include <ghoul/logging/logmanager.h>
+#include <ghoul/opengl/texture.h>
+#include <fstream>
+
+namespace {
+    constexpr std::string_view _loggerCat = "ScreenSpaceTimeVaryingImageOnline";
+
+    constexpr openspace::properties::Property::PropertyInfo JsonFileInfo = {
+        "JsonFilePath",
+        "JSON File Path",
+        "The file path to the JSON data.",
+        openspace::properties::Property::Visibility::User
+    };
```

Comment:
```suggestion
    constexpr openspace::properties::Property::PropertyInfo FileInfo = {
        "FilePath",
        "File Path",
        "The file path to the data containing information about when to display which image.",
        openspace::properties::Property::Visibility::User
    };
```

Mainly to remove the JSON from the property name as a normal user shouldn't need to care what the underlying dataformat actually is.
Naming it `FilePath` also allows us to add additional dataformats later without breaking people's code